WebHostLib/api/downloads.py

```diff
@@ -38,11 +52,12 @@ def get_latest_release(build: str) -> Response:
                 name = "tar"
             elif "exe" in filename:
                 name = "windows"
+            # not currently supported but maybe one day
+            elif "arm64" in filename:
+                name = "arm64"
+            elif "amd64" in filename or "x86_64" in filename:
+                name = "x86_64"
```

Comment:
hm, this overwrites "appimage", right?
What if we have an arm64 and an x86_64 AppImage?